nf-test.config

```diff
@@ -0,0 +1,6 @@
+config {
+    profile "docker"
+    stage {
+        copy "nextflow_schema.json"
+    }
```

Comment:
I'm not sure I follow this

Reply:
I found that nf-test wasn't copying `/nextflow_schema.json` into the test isolation directories under `.nf-test/tests` without this stage directive. Without that, any test that triggers nf-validation would fail due to the missing schema file. The nf-test docs state that only specific things are staged and nextflow_schema.json is not one of them: https://www.nf-test.com/docs/configuration/#file-staging .